virttest/utils_stress.py

```
@@ -0,0 +1,94 @@
+                    if not self.ignore_status:
+                        libvirt.check_exit_status(result)
+            elif "suspend" in event:
+                result = virsh.suspend(vm.name, **dargs)
+                if not self.ignore_status:
+                    libvirt.check_exit_status(result)
+                time.sleep(self.event_sleep_time)
+                result = virsh.resume(vm.name, **dargs)
+                if not self.ignore_status:
+                    libvirt.check_exit_status(result)
+            elif "cpuhotplug" in event:
+                result = virsh.setvcpus(vm.name, self.max_vcpu, "--live",
+                                        **dargs)
+                if not self.ignore_status:
+                    libvirt.check_exit_status(result)
+                    exp_vcpu = {'max_config': self.max_vcpu,
+                                'max_live': self.max_vcpu,
+                                'cur_config': self.current_vcpu,
+                                'cur_live': self.max_vcpu,
+                                'guest_live': self.max_vcpu}
+                    utils_hotplug.check_vcpu_value(vm, exp_vcpu, option="--live")
+                time.sleep(self.event_sleep_time)
+                result = virsh.setvcpus(vm.name, self.current_vcpu, "--live",
+                                        **dargs)
+                if not self.ignore_status:
+                    libvirt.check_exit_status(result)
+                    exp_vcpu = {'max_config': self.max_vcpu,
+                                'max_live': self.max_vcpu,
+                                'cur_config': self.current_vcpu,
+                                'cur_live': self.current_vcpu,
+                                'guest_live': self.current_vcpu}
+                    utils_hotplug.check_vcpu_value(vm, exp_vcpu, option="--live")
+            elif "reboot" in event:
+                vm.reboot()
+                vm.wait_for_login()
+            else:
+                pass
```

please raise the error as event not implemented or not supported, else we it would simply pass the testcase.

---

Sure..